apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -159,6 +172,7 @@ private class HistoryEntry
         public async Task AbortedThrownCorrectly()
         {
             await WriteSampleRowsAsync();
+            LogPerformanceDataBeforeTest();
```

Comment:
Switching to use the beforeafter attribute in xunit -- this should clean things up